scripts/push_docs_to_branch.sh

```diff
@@ -1,11 +1,22 @@
 #!/bin/bash
 
+# In case this script fails in our CI workflows, you can run it locally.
+# For instance, if the doc version for 2.2.10 fails to be pushed, you
+# can do:
+#
+# TMP_DOCS_FOLDER=/tmp/documentation DOCS_BRANCH=documentation GITHUB_REF=refs/tags/2.2.10 ./scripts/push_docs_to_branch.sh
+#
+# This command will push the changes to the `documentation` branch of this repository.
+#
+# Please make sure to have the proper dependencies installed with `make install install-docs`
```

Comment:
Should we make this a numerated list to be more concise? E.g.

1. Run `make install install-docs`
2. Run `make prepare-docs`
3. Run `TMP_DOCS...`

Reply:
yes, that's a great suggestion